### DISCUSSION TITLE
Install Error

### DISCUSSION THREAD

**Discussion by gaiyf** [isce-framework/fringe#81] [Q&A]
There is an cmake ERROR, i am looking forward to  your advice. 

![2023-03-30 09-36-04 的屏幕截图](https://user-images.githubusercontent.com/57431511/228705699-6cd79a61-9d00-43a2-8269-754410c4ae24.png)

**Comment by scottstanie** [comment]
Looks like it couldn't find Python or the Numpy include directories.
Are you using conda (it'll make things easier). If so, then add `-DCMAKE_PREFIX_PATH=$CONDA_PREFIX` to your command after `cmake`
